### DISCUSSION TITLE
--yes equivalent in the config file?

### DISCUSSION THREAD

**Discussion by iacs** [topgrade-rs/topgrade#374] [Q&A] [answered]
I am trying to find if there exists an equivalent of the `--yes` flag in the config file. I want to set it up to auto-yes only flatpak updates, but not the other ones. I haven't been able to find anything documented in the example config. 

`yes = ["flatpak"]` doesn't work and `assume_yes = true` seems to be a global setting affecting every step.

Is it possible to define the equivalend of topgrade --yes flatpak in the config?

**Comment by s34m** [comment]
There is currently no option to do this. I could implement that in the future if you want but it will be not top priority, especially since I sadly currently have not too much time to work on my projects.

**Comment by GideonBear** [comment, marked as answer]
Replaced by #399
